scripts/add-capture_kit-table.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE `organism` (
```

Comment:
Looks like the wrong content for capture kit table.

Could be removed?

Reply:
removed